corehq/apps/locations/static/locations/js/location_types.js

```diff
@@ -321,6 +321,41 @@ hqDefine('locations/js/location_types.js', function(){
             };
         };
     }
+
+    $(function() {
```

Comment:
Something to think about, but probably not immediately actionable. Importing this module now has a side effect (setup on document ready handler). It looks like it didn't have any side effects before, it just exposed an object. Is this additional import/runtime complexity something we should be concerned about? Should try to define some type of distinction between modules that have import side effects and modules that just expose data/functions?